core/types.go

```diff
@@ -32,6 +32,9 @@ type Validator interface {
 	// ValidateState validates the given statedb and optionally the receipts and
 	// gas used.
 	ValidateState(block *types.Block, state *state.StateDB, receipts types.Receipts, usedGas uint64) error
+
+	// TODO: add comments
```

Comment:
can add comments

Reply:
resolved in https://github.com/scroll-tech/go-ethereum/pull/615/commits/acf23b72b5312edadf3ff4a9a1f6a3248ab47d75